src/ICeWall/test/caravel_sky130.ok

```diff
@@ -142,9 +142,11 @@ The NOWIREEXTENSIONATPIN statement will be ignored. See file caravel_sky130/lef/
 [INFO ODB-0134] Finished DEF file: caravel_sky130/chip_io.def
 Loading library data
 Extracting footprint
-[WARNING PAD-0046] No power nets found in design
-[WARNING PAD-0047] No ground nets found in design
-No differences found.
+[WARNING PAD-0046] No power nets found in design.
+[WARNING PAD-0047] No ground nets found in design.
+Differences found at line 4.
+  Type wirebond
+  Type Wirebond
```

Comment:
Looks like there is something that didn't get updated.

Reply:
Thanks for pointing this out. Fixed on 22da9d2